src/wrappers/node.js

```diff
@@ -1,4 +1,4 @@
 if (!global.FormData) {
```

Comment:
Not sure we need the wrapper anymore, that piece of code should just go in smooch.js